h2/src/main/org/h2/engine/Database.java

```diff
@@ -173,6 +173,7 @@
     private Role publicRole;
     private final AtomicLong modificationDataId = new AtomicLong();
     private final AtomicLong modificationMetaId = new AtomicLong();
+    private final AtomicLong remoteSettingsId = new AtomicLong();
```

Comment:
comment here, something like

"Used to trigger the client-side of the jdbc driver to reload some of the settings"

Reply:
2bcbf49f6e8ffcce125c01bee8f8f7b0c4906170